Develop Merge #1

anneramey opened the pull request.

mhowe committed:
Updated object_update parameter id to be object_id, as expected by rest of the handler.

mhowe committed:
Update the max_results parameter id to be page_size to match expected code through handler.

mhowe committed:
Added User Lookup v1 Handler

This hander uses the V1 Cherwell API.

mhowe committed:
Added User Retrieve Handler V2 - uses the Cherwell V2 API with additional additional inputs

mhowe committed:
Removed an extraneous puts statement

anneramey committed:
Added Error Message Handling

mhowe committed:
Added an additional layer of exception checking for how Cherwell responds to errors.

mhowe commented:
Thanks, Anne.  I've approved...

On Thu, Nov 7, 2019 at 11:03 AM anneramey <notifications@github.com> wrote:

> @anneramey <https://github.com/anneramey> requested your review on: #1
> <https://github.com/kineticdata/peripherals-cherwell/pull/1> Develop
> Merge #1 <https://github.com/kineticdata/peripherals-cherwell/pull/1>.
>
> —
> You are receiving this because your review was requested.
> Reply to this email directly, view it on GitHub
> <https://github.com/kineticdata/peripherals-cherwell/pull/1?email_source=notifications&email_token=AAKNEO3SPCDSBWJBZFMQFRTQSRC4XA5CNFSM4JKKCUN2YY3PNVWWK3TUL52HS4DFWZEXG43VMVCXMZLOORHG65DJMZUWGYLUNFXW5KTDN5WW2ZLOORPWSZGOUW2UFHY#event-2780119711>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAKNEO4WOCOOXSYKFCC435TQSRC4XANCNFSM4JKKCUNQ>
> .
>


-- 
Matthew Howe
Technical Consultant

Kinetic Data, Inc.
*Your business. Your process.*

651.556.0924 I www.kineticdata.com | community.kineticdata.com

mhowe committed:
Additional Error Handling added to each handler; Find handler includes encoding modifications.

The object_find handler was erroring when the 'Description' field contained special characters (i.e, enhanced formatting, etc.).  Added encoding to attempt to address this.  Worked in test scenarios.
